spec/solver_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../solver'
+
+describe Solver do
```

Comment:
- [ ] Good job so far 👏  But according to the project requirements The `#factorial` method should have the special case that the factorial of 0 is 1 . Please fix it in order to make your project works as expected 👍

Reply:
well resolved! ✔️